.github/workflows/preDeploy.yml

```diff
@@ -137,3 +137,40 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+  newContributorReminder:
+    runs-on: ubuntu-latest
```

Comment:
For the sake of general efficiency and noise reduction, let's skip this job for `OSBotify`:

```yaml
newContributorWelcomeMessage:
  if: ${{ github.actor != 'OSBotify' }}
  runs-on: ubuntu-latest
```

Reply:
Ok! Small question, I see in the documentation that the `${{ }}` are not needed in `jobs.if` ([doc](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif)). Should I still add it? I can see that in other cases in this file there are `${{ }}`, but in other files there isn't.

Reply:
I guess it's up to you, but I recommend just including it for consistency. That way we don't have to remember when `${{ }}` is required and when it's not.